WikipediaCleaner/src/org/wikipediacleaner/api/APIFactory.java

```diff
@@ -90,10 +91,10 @@ private static HttpClient createHttpClient(HttpConnectionManager manager) {
     HttpClient client = new HttpClient(manager);
     client.getParams().setParameter(
         HttpMethodParams.USER_AGENT,
-        "WPCleaner (+http://en.wikipedia.org/wiki/User:NicoV/Wikipedia_Cleaner/Documentation)");
+        "WPCleaner/" + Version.VERSION + " (+http://en.wikipedia.org/wiki/User:NicoV/Wikipedia_Cleaner/Documentation)");

```

Comment:
Thanks @LDWP .

Can I suggest a few changes ?
* Use String.formatted() instead of concatenation ? (to take advantage of new features available since the code is now built with Java 17 : there's a lot of work to do this on whole code...)
* Change also for https at the same time
* Change also the commented code a few lines below